controllers/nc_news.controllers.js

```diff
@@ -34,18 +40,40 @@ function getArticleById(req, res, next) {
 
 function getCommentsByArticleId(req, res, next) {
   const { article_id } = req.params;
-  fetchCommentsByArticleId(article_id)
-    .then((comments) => {
+  Promise.all([
+    fetchCommentsByArticleId(article_id),
+    checkIfArticleExists(article_id),
+  ])
+    .then(([comments]) => {
       res.status(200).send({ comments });
     })
     .catch((err) => {
       next(err);
     });
 }
 
+function postCommentByArticleId(req, res, next) {
+  const { article_id } = req.params;
+  const postedCommentInfo = req.body;
+
+  Promise.all([
+    checkIfArticleExists(article_id),
```

Comment:
Im not sure we need to check if the article id exists here as we would get a psql error if it didnt whilst trying to add a comment :)

Reply:
Done - thank you :)